README.md

```diff
@@ -1,140 +1,28 @@
-:tada: The **Office UI Fabric React** project has evolved to **Fluent UI**. :tada:
+# Fluent UI Web
 
```

Comment:
We need to have a one or two sentence explanation of what the repo's purpose is here.

> Fluent UI web represents a collection of utilities, React components, and web components for building web applications.